README.es.md

```
@@ -135,6 +135,8 @@ Alternativa: [media/demo.mp4](media/demo.mp4)
 | Wio Tracker L1 Pro            | Gateway de campo portátil          | [Comprar][hw-wio]      |
 | SenseCAP Card Tracker T1000-E | Rastreador compacto                | [Comprar][hw-sensecap] |
 
+> **Host de gateway: [reComputer R1024][hw-r1024]** — No necesitas un Mac Mini. El reComputer R1024 (~$189, ~5 W) es un ordenador edge industrial sin ventilador que ejecuta OpenClaw 24/7. Alimentado por PoE, montable en riel DIN, con doble Ethernet y módulos opcionales LoRa/4G — ideal como gateway siempre encendido en el campo. [Guía de inicio →][wiki-openclaw-recomputer]
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)